README.md

```diff
@@ -2,7 +2,7 @@
 
 OSS Leaderboard shows the top contributors based on their contribution activities on GitHub.
 
-<img src = "https://i.imgur.com/SPMKoJt.png" width = "700">
+<img src = "https://imgur.com/o32dq3s" width = "700">
```

Comment:
Image link is broken, can you replace it with https://i.imgur.com/o32dq3s.png

Reply:
Fixed !

Reply:
Can we use real names here? 

Let's use this as the URL `https://i.imgur.com/o32dq3s.png`

Reply:
ok